netcdf_to_gltf_converter/netcdf/netcdf_data.py

```diff
@@ -68,8 +84,51 @@ def get_data_at_time(self, time_index: int) -> np.ndarray:
         """
         time_filter = {self._time_var.name : time_index}
         return self._data_array.isel(**time_filter).values.flatten()
+    
+    def get_values_at_coordinate(self, coord_index: int) -> np.ndarray:
+        """Get the values for this variable at the specified coordinate index.
 
+        Args:
+            coord_index (int): The coordinate index.
 
+        Returns:
+            np.ndarray: The values at the specified coordinate index.
+        """
+        data_on_coordinate_filter = {
+            self._x_coords_var.dims[0]: coord_index
+        }
+        return self._data_array.isel(data_on_coordinate_filter).values
+    
+    def set_values_at_coordinate(self, coord_index: int, values) -> None:
+        """Set the values for this variable at the specified coordinate index.
+
+        Args:
+            coord_index (int): The coordinate index.
+            values (_type_): The new values.
+        """
+        data_on_coordinate_filter = {
```

Comment:
maybe make it a private property, since it is also used in the method above.

Reply:
Turns out, these were not used and I removed them.